corehq/apps/user_importer/importer.py

```diff
@@ -343,14 +343,17 @@ def get_domain_info(domain, upload_domain, user_specs, domain_info_by_domain, up
         location_cache = SiteCodeToLocationCache(domain)
 
     domain_obj = Domain.get_by_name(domain)
+    if domain_obj is None:
+        raise UserUploadError(_("Domain cannot be set to '{domain}'".format(domain=domain)))
```

Comment:
Nitpick: user-facing text should say "project space" not "domain"...although is the header "domain"? If so, it doesn't really matter.

Reply:
The header is domain and I was wondering if that's what we would actually want? If that is a change we want, I'll leave that for a separate PR

Reply:
Keeping this error sounds good.